cf-testd/cf-testd.c

```diff
@@ -774,6 +774,7 @@ int main(int argc, char *argv[])
     for (int i = 0; i < n_threads; i++)
     {
         thread_configs[i] = (CFTestD_Config*) xmalloc(sizeof(CFTestD_Config));
+        memset(&thread_configs[i], 0, sizeof(CFTestD_Config));
```

Comment:
It's easier to replace the `xmalloc()` call with `xcalloc()`.

Reply:
agreed, will change.